setup/setup-sandbox.sh

```diff
@@ -0,0 +1,27 @@
+OPENSHIFT_CLUSTER_NAME="openshift-cluster"
+OPENSHIFT_CREDENTIALS_NAME="openshift-credentials"
+OPENSHIFT_CONTEXT="openshift-context"
```

Comment:
We must remove references to OpenShift.
Also, we should be aligning both scripts to use similar context names, ie `kbe-minikube` and `kbe-devsandbox`